docs/content/community.md

```diff
@@ -3,6 +3,24 @@ title: Community
 description: Where to find other people, ask for help and share neat stuff with the Porter community
 ---
 
+## Forum
+
+The best place to engage with the community is our [forum]. People can find
+topics that your start and join the conversation, or find an answer to their
```

Comment:
"People can find topics that your start and join the conversation,..." appears to need revising; perhaps some text was lost along the way?

Reply:
Oops, I am going to blame slack for interrupting me mid editing. 😀